shared/chat/conversation/info-panel/menu/index.js

```diff
@@ -65,29 +64,26 @@ const AdhocHeader = (props: AdhocHeaderProps) => (
       />
       {!!props.fullname && <Kb.Text type="BodySmall">{props.fullname}</Kb.Text>}
     </Kb.Box2>
-  </>
+  </Kb.Box2>
 )
 
 type TeamHeaderProps = {
-  teamname: string,
+  isMuted: boolean,
   memberCount: number,
+  teamname: string,
 }
 const TeamHeader = (props: TeamHeaderProps) => (
-  <>
-    <Kb.Avatar
-      size={Styles.isMobile ? 64 : 48}
-      teamname={props.teamname}
-      style={Kb.avatarCastPlatformStyles(styles.headerAvatar)}
-    />
+  <Kb.Box2 direction="vertical" gap="tiny" gapStart={false} gapEnd={true} style={styles.headerContainer}>
+    <TeamAvatar teamname={props.teamname} isMuted={props.isMuted} isSelected={false} isHovered={false} />
     <Kb.Box2 direction="vertical" centerChildren={true}>
       <Kb.Text type="BodySemibold">{props.teamname}</Kb.Text>
       <Kb.Text type="BodySmall">{`${props.memberCount} member${props.memberCount !== 1 ? 's' : ''}`}</Kb.Text>
     </Kb.Box2>
-  </>
+  </Kb.Box2>
 )
 
 class InfoPanelMenu extends React.Component<Props> {
-  componentDidDUpdate(prevProps: Props) {
+  componentDidUpdate(prevProps: Props) {
```

Comment:
This typo was a bug in the existing code!  Checked that everything works with it fixed.